project_role/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     'name': 'Project Roles',
-    'version': '12.0.1.1.0',
+    'version': '12.0.1.1.1',
```

Comment:
There's no need anymore of manual version bumping FYI

Reply:
Cool! How it's decided if that's major minor or fix?

Reply:
https://odoo-community.org/blog/the-oca-blog-1/post/the-oca-github-bot-can-now-merge-pull-requests-92